nbcparser/main.go

```diff
@@ -1,40 +1,36 @@
 package main
 
 import (
-	"log"
```

Comment:
did you want to actually switch over to `fmt`? personally I'd prefer sticking with `log`

Reply:
switched back to log